libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeLegend/TreeLegend.tsx

```diff
@@ -60,35 +59,6 @@ export class TreeLegend extends React.Component<ITreeLegendProps> {
                 </div>
               </Stack>
             </Stack>
-            <svg
```

Comment:
There is a mis communication we still need that big grey dot.
but just change the mouse cursor style, from that hand pointer to regular mouse arrow since it is not clickable.